<launch>
    <arg name="ugv_id" default="1" />
    <arg name="ugv_name" default="ugv" />
    <node pkg="sunray_ugv_control" type="ugv_terminal_control" name="terminal_control" output="screen">
        <param name="ugv_id" value="$(arg ugv_id)" />
        <param name="ugv_name" value="$(arg ugv_name)" />
    </node>
</launch>